cmd/kubectl-accurate/sub/sub_create.go

```diff
@@ -36,6 +38,8 @@ This effectively creates a namespace named NAME as a sub-namespace of NS.`,
 		},
 	}
 
+	cmd.Flags().StringToStringVar(&opts.labels, "labels", opts.labels, "the labels to be propagated to the sub-namespace")
```

Comment:
Please add an example of the usage to the description.  e.g.,
```suggestion
	cmd.Flags().StringToStringVar(&opts.labels, "labels", opts.labels, "the labels to be propagated to the sub-namespace.  Example: a=b,c=d")
```

Reply:
added.